src/Uno.UI.Runtime.Skia.Win32/Rendering/Win32WindowWrapper.Rendering.cs

```diff
@@ -55,7 +55,6 @@ private void Render()
 		{
 			var nativeElementClipPath = ((CompositionTarget)((IXamlRootHost)this).RootElement!.Visual.CompositionTarget!).OnNativePlatformFrameRequested(_surface?.Canvas, size =>
 			{
-				_renderer.Reset();
```

Comment:
It does not need to reset on normal window size changes/maximize?

Reply:
I removed `Reset` entirely. It was only being called right before UpdateSize when the size of the window changes, so I moved the logic inside UpdateSize.